tegra-nouveau.xml

```diff
@@ -5,12 +5,12 @@
   <remote name="baserock" fetch="http://git.baserock.org/git/"/>
   <remote name="nvidia" fetch="https://github.com/NVIDIA/"/>
   <remote name="gnurou" fetch="https://github.com/Gnurou/"/>
-  <remote name="denvit" fetch="https://github.com/denysvitali/"/>
+  <remote name="samt" fetch="https://github.com/Samt43/"/>
```

Comment:
I think you forgot to leave my remote, by simply replacing my name with yours the tegra-rootfs-scripts repo is going to fail because there is no "denvit" remote